build.sh

```diff
@@ -29,3 +29,9 @@ docker run -v "$(pwd)/Generated:/app/Generated" \
 if [ "$BTCPAYGEN_REVERSEPROXY" == "nginx" ]; then
     cp Production/nginx.tmpl Generated/nginx.tmpl
 fi
+
+if [ "$BTCPAYGEN_REVERSEPROXY" == "traefik" ]; then
+    cp Traefik/traefik.toml Generated/traefik.toml
+    :> Generated/acme.json
```

Comment:
what is `:>`? a smiley?

Reply:
That one ensures that there is an empty file at `Generated/acme.json`